docs/templates/plugin.rst.j2

```diff
@@ -142,7 +142,7 @@ Parameters
                                 {% elif choice is sameas false %}
                                     {% set choice = 'no' %}
                                 {% endif %}
-                                {% if (value.default is string and value.default == choice) or (value.default is iterable and value.default is not string and choice in value.default) %}
+                                {% if (value.default is not list and value.default == choice) or (value.default is list and choice in value.default) %}
```

Comment:
I'm not sure whether it will work with Jinja syntax, but I'd do smth like:
```jinja
{% if choice in (value.default if value.default is list else [value.default]) %}
```

Reply:
The last time I tried, array literals did't work in Jinja (unfortunately).